src/objc/messages.rs

```diff
@@ -249,6 +399,46 @@ where
 /// See also [msg_class], if you want to send a message to a class.
 #[macro_export]
 macro_rules! msg {
+    [$env:expr; $receiver:tt $name:ident $(: $arg1:tt)?
+                             $($namen:ident: $argn:tt)* $(,$va_argn:ident)*] => {
+        {
+            let sel = $crate::objc::selector!($($arg1;)? $name $(, $namen)*);
+            let sel = $env.objc.lookup_selector(sel)
+                .expect("Unknown selector");
+            if cfg!(debug_assertions)
+            {
```

Comment:
Nitpick: this should be on the previous line so it doesn't look like it's a separate block.